cli/lib/kontena/cli/nodes/list_command.rb

```diff
@@ -67,7 +66,7 @@ def node_data
         unless quiet?
           grid_health = grid_health(grid, grid_nodes)
           grid_nodes.each do |node|
-            node['health_icon'] = health_icon(node_health(node, grid_health))
+            node['name'] = health_icon(node_health(node, grid_health)) + " " + node['name']
```

Comment:
The "blank" health icon column was there to offset the headers so that it looks like:
```
  FOOFOO FOO
* foofoo
? foofoo
```

instead of

```
FOOFOO FOO
* foofoo
? foofoo
```